src/property-filter/filtering-token/index.tsx

```diff
@@ -315,6 +324,7 @@ function TokenDismissButton({
       aria-label={ariaLabel}
       onClick={onClick}
       disabled={disabled}
+      {...getAnalyticsMetadataAttribute({ action: 'dismiss' })}
```

Comment:
can this action be more specific, like: "tokenRemove"?

Reply:
I prefer to keep it aligned with Multiselect and Tokengroup